nixos/modules/services/web-apps/ethercalc.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ethercalc;
+in {
+  options = {
+    services.ethercalc = {
+      enable = mkEnableOption "ethercalc, an online collaborative spreadsheet server (persistent state will be maintained under <filename>/var/lib/ethercalc</filename>)";
+
+      package = mkOption {
+        default = pkgs.ethercalc;
```

Comment:
This software is still being maintained right? I noticed there hasn't been a commit in over a year.

Reply:
Right, it didn't have commits in over a year, but the project is still active in the issue tracker. It is a good and stable piece of software which doesn't really need new features. However, I am a bit concerned regarding version bumps in the lock file, because they won't happen. Perhaps we should indicate this somewhere.

Reply:
OK cool. As long as they are shipping required security updates we're good :+1: Thanks!